src/Categories/Morphism/Reasoning/Core.agda

```diff
@@ -5,11 +5,16 @@ open import Categories.Category
   Helper routines most often used in reasoning with commutative squares,
   at the level of arrows in categories.
 
-  Basic  : reasoning about identity
+  Identity : reasoning about identity
+  Assoc4   : associativity short-cuts for 4-term
```

Comment:
I'm not sure this line makes grammatical sense

Reply:
You are right -- will fix.